specification/consumption/resource-manager/Microsoft.Consumption/stable/2018-01-31/examples/MarketplacesList.json

```diff
@@ -0,0 +1,42 @@
+{
+  "parameters": {
+    "api-version": "2018-01-31",
+    "scope": "subscriptions/subid"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/subscriptions/subid/providers/Microsoft.Consumption/marketplaces/marketplaceId1",
+            "name": "usageDetailsId1",
+            "type": "Microsoft.Consumption/marketplaces",
+            "tags": {
+              "env": "newcrp",
+              "dev": "tools"
+            },
+            "properties": {
+              "billingPeriodId": "/subscriptions/subid/providers/Microsoft.Billing/billingPeriods/201702",
+              "usageStart": "2017-02-13T00:00:00Z",
+              "usageEnd": "2017-02-13T23:59:59Z",
+              "instanceName": "shared1",
+              "instanceId": "/subscriptions/subid/resourceGroups/Default-Web-eastasia/providers/Microsoft.Web/sites/shared1",
+              "currency": "USD",
+              "consumedQuantity": 0.00328,
+              "pretaxCost": 0.67,
+              "isEstimated": false,
+              "offerName": "offer1",
```

Comment:
So this case it is not an OAV bug. According to the swagger, it looks like this should be `offerType` instead